ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/sno/baselinecaps/none/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-sno-baselinecaps-none-provision-chain.yaml

```diff
@@ -0,0 +1,38 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-disconnected-private-sno-baselinecaps-none-provision
+  steps:
+    - ref: aws-provision-vpc-disconnected
```

Comment:
I guess we can directly reference `cucushift-installer-rehearse-aws-ipi-disconnected-private-sno-provision` here if no difference with the steps.

Reply:
good idea, no difference, updated.